src/pages/AboutUs.jsx

```diff
@@ -108,7 +109,7 @@ function RyazMethod({ isNavVisible }) {
       ) : (
         <div className="top" />
       )}
-      <div className="about-section">
+      {/* <div className="about-section">
         <h3>Our Team</h3>
```

Comment:
Okay so we removing the "Our Team" section?

Reply:
This is included in the changes